htdocs/commande/card.php

```diff
@@ -2628,7 +2628,7 @@
 				}
 
 				// Cancel order
-				if ($object->statut == Commande::STATUS_VALIDATED && (!empty($usercanclose) || !empty($usercancancel))) {
+				if ($object->statut == Commande::STATUS_VALIDATED && !empty($usercancancel)) {
```

Comment:
Why removing the   !empty($usercanclose) 
It is a or with the condition   !empty($usercancancel)
so only one of this 2 conditions should be necessary, as expected.

Reply:
If a user does not have the right to cancel why can he do it if he has the right to close?

Security error!

We do not give the right to delete but he can do it!

Reply:
Why can a user who has the right to close cancel when there is an authorization defined for this?

I do not understand! For me it's a security error!